docs/framework/20categories/20architecture/liquidity-networks.md

```diff
@@ -1,3 +1,24 @@
 ### Liquidity Networks
+Liquidity networks enable users to transfer already minted assets from one chain to another. With such protocols, there's no ‘bridging’ of assets, i.e., no assets move across chains. Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap. For example, a user receives the desired asset on the destination network via the LP. In return, the LP receives the user’s assets on the source network and fees for providing the desired asset. 
```

Comment:
"one user.... another bridge user....". I think they are both bridge users. Maybe remove the word bridge.

Reply:
Should the ordering be, the user and the LP, having the LP second in the sentence. This will allow the reader to more easily understand what is happening: a user wants to move value across chains, hence it is the user who wants to do the action, not the LP

Reply:
I would change from, "Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap.", to, "Instead, assets are exchanged between a user and a Liquidity Provider (LP), resulting in a cross-chain swap."